src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] result;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            StringBuilder allFile = new StringBuilder();
+            String value = reader.readLine();
+            if (value == null || value.isEmpty()) {
+                return new String[0];
+            }
+            while (value != null) {
+                if (!value.isEmpty()) {
+                    allFile.append(value);
+                    allFile.append(" ");
+                }
+                value = reader.readLine();
+            }
+            String[] strArray = allFile.toString().toLowerCase().split(" ");
```

Comment:
starting from here, you are no more reading from the file
let's move the following code out of the `try` block

Reply:
done